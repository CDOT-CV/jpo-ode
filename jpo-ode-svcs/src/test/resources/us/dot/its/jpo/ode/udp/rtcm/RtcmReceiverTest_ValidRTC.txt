001c6d25244028d2d4b29bc25ec0c12c418d300133ed980037bdd1a80c6358121dd4e499ffc6712e91f0d10f4c00f90266005105115939553131053951153939048081393d391400003551492535093114810531313d64513985d880d4b8d0d080bc8109bdbdd080d4b8d0d0023197e4600180000243d35cf767e1000243d36ed90ae181010100030180c620fb90caad3b9c5082082b11e72ccf15fc32396921000325fbf96e83279c830101800348010780032040958005000001e040800320409780050080012040800182800500800130408001838005008001f040000187000126000180818280e9acd3deb91a1c3f8c3fac2e1bd37772c16ffefc1a51785d05b494a8f9c7628082e2fbe39a6fbdb7ce9a5fa234570604b3628493c89ec1c89edec5132e25a017d84588eebab47f4756a80d24f339c1b139298abfc6ece497c5de5df93beaa60c98