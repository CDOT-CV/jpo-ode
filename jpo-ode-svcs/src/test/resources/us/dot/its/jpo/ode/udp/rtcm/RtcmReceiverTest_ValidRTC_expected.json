{
  "metadata": {
    "securityResultCode": "success",
    "recordGeneratedBy": "RSU",
    "schemaVersion": 9,
    "sanitized": false,
    "serialId": {
			"streamId": "609a6611-8a14-4ca7-a6a2-44f68c163c7d",
			"bundleSize": 1,
			"bundleId": 0,
			"recordId": 0,
			"serialNumber": 0
		},
    "recordType": "rtcmTx",
    "maxDurationTime": 0,
    "source": "RSU",
    "isCertPresent": false,
    "payloadType": "us.dot.its.jpo.ode.model.OdeAsn1Payload",
    "asn1": "001C6D25244028D2D4B29BC25EC0C12C418D300133ED980037BDD1A80C6358121DD4E499FFC6712E91F0D10F4C00F90266005105115939553131053951153939048081393D391400003551492535093114810531313D64513985D880D4B8D0D080BC8109BDBDD080D4B8D0D0023197E4600180000243D35CF767E1000243D36ED90AE181010100030180C620FB90CAAD3B9C5082082B11E72CCF15FC32396921000325FBF96E83279C830101800348010780032040958005000001E040800320409780050080012040800182800500800130408001838005008001F040000187000126000180818280E9ACD3DEB91A1C3F8C3FAC2E1BD37772C16FFEFC1A51785D05B494A8F9C7628082E2FBE39A6FBDB7CE9A5FA234570604B3628493C89EC1C89EDEC5132E25A017D84588EEBAB47F4756A80D24F339C1B139298ABFC6ECE497C5DE5DF93BEAA60C98",
    "odeReceivedAt": "2024-11-26T23:53:21.120Z",
    "originIp": "127.0.0.1"
  },
  "payload": {
    "data": {"bytes": "001C6D25244028D2D4B29BC25EC0C12C418D300133ED980037BDD1A80C6358121DD4E499FFC6712E91F0D10F4C00F90266005105115939553131053951153939048081393D391400003551492535093114810531313D64513985D880D4B8D0D080BC8109BDBDD080D4B8D0D0023197E4600180000243D35CF767E1000243D36ED90AE181010100030180C620FB90CAAD3B9C5082082B11E72CCF15FC32396921000325FBF96E83279C830101800348010780032040958005000001E040800320409780050080012040800182800500800130408001838005008001F040000187000126000180818280E9ACD3DEB91A1C3F8C3FAC2E1BD37772C16FFEFC1A51785D05B494A8F9C7628082E2FBE39A6FBDB7CE9A5FA234570604B3628493C89EC1C89EDEC5132E25A017D84588EEBAB47F4756A80D24F339C1B139298ABFC6ECE497C5DE5DF93BEAA60C98"},
    "dataType": "us.dot.its.jpo.ode.model.OdeHexByteArray"
  }
}